### PR TITLE
#8 - Add flip cards

### LINKED ISSUE #8: Add double-faced cards as tokens?
Would it make sense (and be possible) to use the Token system to manage flip cards?

An example would be Huntmaster of the fells && Ravager of the Fells.

A way to get around the unique name problem is adding a space at the end of the name (already the case for alternative tokens).

### PR DESCRIPTION
**Pull request by ColinHebert**
Fixes #8 

Added all the double-faced and flip cards to the token list. This way it's possible to flip without having to type the name, without having the card in the sideboard and by simply using the token shortcuts.

The double-faced cards are using the original art from gatherer, the suffix " (flip)" has been added to avoid collisions with the cards with the same name already coming from Oracle.

The flip cards are using the art for magic art as the flipped art from gatherer can't be obtained easily (the URL can't be used by Cockatrice).
The suffix " (flip)" has been added to avoid collisions with the cards with the same name.

Only the flip/double-faced cards up to Origin are added, the new planeswalkers from ORI aren't available yet.

### CODE REVIEW

**Comment by ColinHebert** [comment]
At the moment some of those tokens are a bit wordy. Hopefully that will be fixed with Cockatrice/Cockatrice#241

**Comment by ColinHebert** [comment]
Added actual flipcards (BOK/SOK/CHK)

**Comment by ColinHebert** [comment]
Nevermind, it's been added to Cockatrice itself: https://github.com/Cockatrice/Cockatrice/releases

**Comment by Psithief** [comment]
Thanks for the spam. :-1:
